packages/clerk-js/src/core/clerk.ts

```diff
@@ -949,7 +954,29 @@ export default class Clerk implements ClerkInterface {
     this.#componentControls?.updateProps(props);
   };
 
-  #loadInStandardBrowser = async (): Promise<void> => {
+  #syncWithPrimary = () => {
+    const q = new URLSearchParams({
+      redirect_url: `${window.location.href}?synced=true`,
```

Comment:
:wrench: @panteliselef I think we missed this one!

Reply:
Yes, already fixed :)